astropy/modeling/tests/test_constraints.py

```diff
@@ -390,6 +391,10 @@ def test_fit_with_fixed_and_bound_constraints(fitter):
     stay within their given constraints.
     """
 
+    # DogBoxLSQFitter causes failure on s390x, aremel possibly others (not x86_64 or arm64)
+    if fitter == fitting.DogBoxLSQFitter and (platform.machine() not in ('x86_64', 'arm64')):
+        pytest.xfail()
```

Comment:
Can you give this xfail a reason? It could be as simple as referring to the original issue so we have a trail. Thanks!

Reply:
See 495cbf6.